commands/packageCommand.mjs

```diff
@@ -0,0 +1,314 @@
+                }
+
+                case "clear": {
+                    _handleClear();
+                    break;
+                }
+
+                case "unpack": {
+                    await _handleUnpack(argv);
+                    break;
+                }
+
+                case "pack": {
+                    await _handlePack(argv);
+                    break;
+                }
+
+                default: {
+                    if ( !currentPackageId ) {
+                        console.error(chalk.red("No package ID is currently set. Use `package workon <id>` to set it."));
+                        return;
+                    }
+                    console.log(`Currently in ${chalk.magenta(currentPackageType)} ${chalk.cyan(currentPackageId)}`);
+                    break;
+                }
+            }
+        }
+    }
+
+    /* -------------------------------------------- */
+
+    /**
+     * Set the current package ID and type
+     * @param {Object} argv                  The command line arguments
+     * @private
+     */
+    function _handleWorkon(argv) {
+        if ( argv.value ) {
+            currentPackageId = argv.value;
+        }
+        Config.instance.set("currentPackageId", currentPackageId);
+
+        if ( !argv.type ) {
+            const game = discoverPackageDirectory(argv);
+            const pkgCount = game.packages.filter(p => p[0] === currentPackageId).length;
+            if ( pkgCount > 1 ) {
+                console.error(chalk.red(`Multiple packages with ID ${chalk.cyan(currentPackageId)} found. Please specify the package type with ${chalk.yellow("--type")}`));
+                return;
+            }
+            const pkg = game.worlds.get(currentPackageId) ?? game.systems.get(currentPackageId) ?? game.modules.get(currentPackageId);
+            if ( !pkg ) {
+                console.error(chalk.red(`No package with ID ${chalk.cyan(currentPackageId)} found.`));
+                return;
+            }
+            currentPackageType = pkg.type;
+        }
+
+        Config.instance.set("currentPackageType", currentPackageType);
+        console.log(`Swapped to ${chalk.magenta(currentPackageType)} ${chalk.cyan(currentPackageId)}`);
+    }
+
+    /* -------------------------------------------- */
+
+    /**
+     * Clear the current package ID and type
+     * @private
+     */
+    function _handleClear() {
+        currentPackageId = null;
+        currentPackageType = null;
+        Config.instance.set("currentPackageId", currentPackageId);
+        Config.instance.set("currentPackageType", currentPackageType);
+        console.log("Cleared current Package");
+    }
+
+    /* -------------------------------------------- */
+
+    /**
+     * Discover the list of all Packages in the dataPath
+     * @param {Object} argv      The command line arguments
+     * @returns {Object}         An object containing discovered packages
+     */
+    function discoverPackageDirectory(argv) {
```

Comment:
Since this function and the one below aren't exported, it seems they might as well be prefixed with an underscore, since they can't be called from anywhere else anyway.